### PR TITLE
Adding advisories for oC-SA-2016-006, oC-SA-2016-007, oC-SA-2016-008,…

### PR DESCRIPTION
**Pull request by DeepDiver1975**
… oC-SA-2016-009 and oC-SA-2016-010

### CODE REVIEW

**Comment by PVince81** [comment]
Looks fine :+1:

File: advisories/oc-sa-2016-008.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<li><a href="https://github.com/owncloud/core/commit/830a080f0e4b67d6265ba09ab578060945f28916">core/830a080f0e4b67d6265ba09ab578060945f28916</a></li>
+</ul>
+
+        </ul>
+        <h3>Action Taken</h3>
+        <p>ownCloud is now preventing files that being with 'file://' from being parsed.</p>
+        <h3>Acknowledgements</h3>
+        <p>The ownCloud team thanks the following people for their research and responsible disclosure of the above advisory:</p>
+        <ul>
+
```
Comment (RealRancor) [inline advisories/oc-sa-2016-008.php]: Who?

File: advisories/oc-sa-2016-009.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<li><a href="https://github.com/owncloud/core/commit/b375086c473f7117d6ee8b61cf0c6e24350b113a">core/b375086c473f7117d6ee8b61cf0c6e24350b113a</a></li>
+</ul>
+
+        </ul>
+        <h3>Action Taken</h3>
+        <p>The storage code has been reviewed and been patched to properly setup the storage. Furthermore several hardenings have been added to ownCloud which will highly reduce the change of a successful exploitation of similar vulnerabilities in the future.</p>
+        <h3>Acknowledgements</h3>
+        <p>The ownCloud team thanks the following people for their research and responsible disclosure of the above advisory:</p>
+        <ul>
+
```
Comment (RealRancor) [inline advisories/oc-sa-2016-009.php]: Who?

**Comment by RealRancor** [comment]
advisories/oc-sa-2016-007.php looks empty

**Comment by PVince81** [comment]
@DeepDiver1975 @danimo edit ?

**Comment by danimo** [comment]
@DeepDiver1975 Status?

File: advisories/oc-sa-2016-008.php (hunk 1 above)
Comment (LukasReschke) [inline advisories/oc-sa-2016-008.php]: Me.

File: advisories/oc-sa-2016-009.php (hunk 2 above)
Comment (LukasReschke) [inline advisories/oc-sa-2016-009.php]: Me

**Pull request closed without merging**